### PR TITLE
Add an initial doctrine migration

### PR DESCRIPTION
**Pull request by aaa2000**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | ?
| Documentation | no
| Translation   | no
| Fixed tickets | 
| License       | MIT

Add an initial doctrine migration to simplify the make file in https://github.com/wallabag/wallabag/pull/3398

If there are already migration executed, this initial version is marked migrated else the migration execute to create initial tables

- [x] Implements the `down` method

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,10 +72,6 @@ script:
     - travis_wait bash composer install -o  --no-interaction --no-progress --prefer-dist
     - make prepare DB=$DB
 
-    - echo "travis_fold:start:migrations"
```
Comment (j0k3r) [inline .travis.yml]: `travis_fold` is really cool to avoid noise lines on travis log.
Maybe you can put them around make prepare?

File: Makefile [hunk 2]
```diff
@@ -30,18 +30,7 @@ ifdef DB
 endif
 	-php bin/console doctrine:database:drop --force --env=test
 	php bin/console doctrine:database:create --env=test
-ifndef DB ## make test does not define DB
```
Comment (Kdecherf) [inline Makefile]: Please keep in mind that some migrations don't support SQLite (`Version20161214094402`, `Version20170511211659`). So we could have inconsistencies with `make test`

Comment (aaa2000) [reply Makefile]: There was 2 migrations not executed correctly `php bin/console doctrine:migrations:migrate ` and I need modify it, but `Version20161214094402` and `Version20170511211659` were OK, it seems that only the `down` doesn't support SQLITE

Comment (aaa2000) [reply Makefile]: But I don't understand why in local SQLite migration was not OK, but with travis it was OK

Comment (aaa2000) [reply Makefile]: Ok in travis the versions not correct in local was skipped

```
  ++ migrating 20170824113337

  SS skipped (Reason: It seems that you already played this migration.)
  
  ++ migrating 20160812120952

  SS skipped (Reason: It seems that you already played this migration.)
```

File: app/DoctrineMigrations/Version20160812120952.php [hunk 3]
```diff
@@ -30,7 +30,20 @@ public function up(Schema $schema)
         $clientsTable = $schema->getTable($this->getTable('oauth2_clients'));
         $this->skipIf($clientsTable->hasColumn('name'), 'It seems that you already played this migration.');
 
-        $clientsTable->addColumn('name', 'blob');
+        if ('sqlite' === $this->connection->getDatabasePlatform()->getName()) {
+            // Can't use $clientsTable->addColumn('name', 'blob');
+            // because of the error:
+            // SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL
+            $databaseTablePrefix = $this->container->getParameter('database_table_prefix');
+            $this->addSql('CREATE TEMPORARY TABLE __temp__' . $databaseTablePrefix . 'oauth2_clients AS SELECT id, random_id, redirect_uris, secret, allowed_grant_types FROM ' . $databaseTablePrefix . 'oauth2_clients');
```
Comment (nicosomb) [inline app/DoctrineMigrations/Version20160812120952.php]: 🤢

Comment (aaa2000) [reply app/DoctrineMigrations/Version20160812120952.php]: In fact, I have used `doctrine:migrations:diff` to generate the SQLITE migration, I had to modify it to use the `database_table_prefix`

**Comment by Kdecherf** [comment]
Does this PR ready for review?

**Comment by aaa2000** [comment]
Yes

**Pull request merged**